api/librdb-api.h

```diff
@@ -202,14 +203,15 @@ typedef struct RdbHandlersDataCallbacks {
     RdbRes (*handleStringValue)(RdbParser *p, void *userData, RdbBulk str);
     RdbRes (*handleListElement)(RdbParser *p, void *userData, RdbBulk str);
 
-    /*** TODO: RdbHandlersDataCallbacks: handleHashElement, handleSetElement, handleZsetElement, handleStreamElement, handleStreamMetadata, handleStreamCgroup  ***/
+    /*** TODO: RdbHandlersDataCallbacks: handleHashElement, handleSetElement, handleZsetElement ***/
     RdbRes (*handleHashElement)(RdbParser *p, void *userData, RdbBulk field, RdbBulk elm, uint64_t totalNumElm);
     RdbRes (*handleSetElement)(RdbParser *p, void *userData, RdbBulk elm, uint64_t totalNumElm);
     RdbRes (*handleZsetElement)(RdbParser *p, void *userData, RdbBulk elm, double score, uint64_t totalNumElm);
-    RdbRes (*handleStreamElement)(RdbParser *p, void *userData, streamID *id, RdbBulk field, RdbBulk value, uint64_t totalNumElm);
-    RdbRes (*handleStreamMetadata)(RdbParser *p, void *userData, uint64_t len, uint64_t entriesAdded, streamID *firstID, streamID *lastID, streamID *maxDeletedEntryID);
-    RdbRes (*handleStreamCgroup)(RdbParser *p, void *userData, streamCG *cgroup); /* when succeeds, 'cgroup' belongs to the calee  */
-
+    /*** TODO: RdbHandlersDataCallbacks: handleStreamNewGroup, handleStreamMetadata, handleStreamElement, handleStreamConsumerPendingEntry ***/
+    RdbRes (*handleStreamNewGroup)(RdbParser *p, void *userData, RdbBulk grpName, RdbStreamID lastId);
+    RdbRes (*handleStreamMetadata)(RdbParser *p, void *userData, RdbStreamMeta *meta);
+    RdbRes (*handleStreamElement)(RdbParser *p, void *userData, RdbStreamID *id, RdbBulk field, RdbBulk value, uint64_t totalNumElm);
+    RdbRes (*handleStreamConsumerPendingEntry)(RdbParser *p, void *userData, RdbBulk groupName, RdbStreamPendingEntry *pendingEntry);
```

Comment:
most users will probably only want to ever listen only to the consumer PEL (if any).
but someone (e.g. redis), may wanna get the group PEL too..
since it's encoded in the rdb, i guess we need a CB for it.